src/DFTK.jl

```diff
@@ -39,6 +39,7 @@ include("common/zeros_like.jl")
 include("common/norm.jl")
 include("common/quadrature.jl")
 include("common/hankel.jl")
+include("common/hydrogenic.jl")
```

Comment:
(It would be nice if these lines were sorted to reduce merge conflicts between PRs that append files.)

Reply:
You mean within each block. That might not be a bad idea.